include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -340,6 +340,18 @@ def TT_CatOp : TT_Op<"cat", [NoMemoryEffect,
     let assemblyFormat = "$lhs `,` $rhs attr-dict `:` functional-type(operands, results)";
 }
 
+// TODO: Add verifier.  Operand shapes + encodings must be the same, and output
+// shape is also constrained.
+def TT_StackMinorOp : TT_Op<"stack_minor", [NoMemoryEffect, SameOperandsAndResultElementType]> {
```

Comment:
Can we have Experimental prefix here? This won't be part of 3.0 release

Reply:
I see. Basically `axis` can only be "-1" for now?